.github/workflows/base-release.yaml

```diff
@@ -115,10 +117,14 @@ jobs:
         if: inputs.distribution == 'otelcol-contrib'
         run: mv distributions/otelcol-contrib/dist/**/* distributions/otelcol-contrib/artifacts/
 
-      - name: Show built or downloaded content
-        if: inputs.distribution == 'otelcol-contrib'
+      - name: Show built or downloaded content (Linux)
+        if: inputs.distribution == 'otelcol-contrib' && runner.os != 'Windows'
         run: ls -laR distributions/otelcol-contrib/artifacts
 
+      - name: Show built or downloaded content (Windows)
```

Comment:
Have you tried `shell: bash` instead of using two different actions?

Reply:
Good point. I didn't try it out with `bash`. I'll check whether it works, because indeed having repeated steps for different OSes doesn't feel good.

Reply:
@pjanotti I could apply this suggestion in some parts of the workflow. 

In some steps, like `make generate-sources`, it didn't work. For instance, in this specific example it wasn't being able to find the `ocb` binary.

But it already saved us some duplication and conditionals.